src/Dapr.PluggableComponents/Components/SecretStore/SecretStoreResponse.cs

```diff
@@ -0,0 +1,40 @@
+﻿// ------------------------------------------------------------------------
+// Copyright 2023 The Dapr Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------------------------------------------
+
+using Dapr.PluggableComponents.Utilities;
+using Dapr.Proto.Components.V1;
+using Google.Protobuf;
+
+namespace Dapr.PluggableComponents.Components.SecretStore;
+
+/// <summary>
+/// Represents properties associated with a response to retrieving bulk secret from a secret store.
+/// </summary>
+public sealed record SecretStoreResponse
```

Comment:
Is this type used anywhere?

Reply:
It was used in the bulk response.  Looking at that again, it seems like we could just reuse the individual get response type; they're functionally identical.  (There's a slight risk of future API changes causing the two gRPC types to vary; if that ever happens we might have to reintroduce separate types which would be a breaking change. 
 We can just cross that road if/when it comes.)